docs/project/list-of-diagnostics.md

```diff
@@ -263,7 +263,7 @@ The diagnostic id values reserved for .NET Libraries analyzer warnings are `SYSL
 |  __`SYSLIB1222`__ | Constructor annotated with JsonConstructorAttribute is inaccessible. |
 |  __`SYSLIB1223`__ | Attributes deriving from JsonConverterAttribute are not supported by the source generator. |
 |  __`SYSLIB1224`__ | Types annotated with JsonSerializableAttribute must be classes deriving from JsonSerializerContext. |
-|  __`SYSLIB1225`__ | *`SYSLIB1220`-`SYSLIB1229` reserved for System.Text.Json.SourceGeneration.* |
+|  __`SYSLIB1225`__ | Type includes ref like property, field or constructor parameter. |
```

Comment:
I made this comment to discuss if "byref-like" is more appropriate, including its usage in error/warning messages.

Reply:
In the docs we seem to be using "ref struct", "ref like" and "byref like" interchangeably. @stephentoub @terrajobst any preferences on naming?

Reply:
"ref like" and "byref like" seem most appropriate because it's language neutral, which is what we usually prefer in docs. Between the two I don't have a preference because we seem to use both quite a bit.